### PR TITLE
Update Chronos Prophet Base Model

### PR DESCRIPTION
**Pull request by PeterQiu0516**
Update Chronos Prophet Base Model

### CODE REVIEW

**Comment by PeterQiu0516** [comment]
Jenkins passed: http://10.239.176.111:18888/view/ZOO-PR/job/ZOO-PR-Validation/5984/

File: pyzoo/zoo/chronos/model/forecast/prophet_forecaster.py [hunk 1]
```diff
@@ -124,7 +120,9 @@ def evaluate(self, validation_data, metrics=['mse']):
         if self.internal.model is None:
             raise RuntimeError(
                 "You must call fit or restore first before calling evaluate!")
-        return self.internal.evaluate(None, validation_data, metrics=metrics)
+        return self.internal.evaluate(target=validation_data,
+                                      data=None,
```
Comment (shane-huang) [inline pyzoo/zoo/chronos/model/forecast/prophet_forecaster.py:124]: just remove "data=None"

File: pyzoo/test/zoo/chronos/model/test_prophet.py [hunk 2]
```diff
@@ -36,54 +36,61 @@ def setup_method(self, method):
...
+        evaluate_result = self.model.fit_eval(data=self.data,
+                                              validation_data=self.validation_data,
+                                              **self.config)
         # test predict
         result = self.model.predict(horizon=self.horizon)
         assert result.shape[0] == self.horizon
         # test evaluate
-        evaluate_result = self.model.evaluate(x=None, target=self.target, metrics=['mae', 'smape'])
+        evaluate_result = self.model.evaluate(target=self.validation_data,
+                                              data=None,
```
Comment (shane-huang) [inline pyzoo/test/zoo/chronos/model/test_prophet.py:62]: just remove data=None

File: pyzoo/test/zoo/chronos/model/test_prophet.py [hunk 3]
```diff
@@ -36,54 +36,61 @@ def setup_method(self, method):
...
+            self.model.evaluate(target=self.validation_data, data=1)
 
         with pytest.raises(ValueError, match="Input invalid target of None"):
-            self.model.evaluate(x=None, target=None)
+            self.model.evaluate(target=None, data=None)
 
         with pytest.raises(Exception,
                            match="Needs to call fit_eval or restore first before calling predict"):
-            self.model.predict(x=None)
+            self.model.predict(data=None)
```
Comment (shane-huang) [inline pyzoo/test/zoo/chronos/model/test_prophet.py:78]: remove data=None

File: pyzoo/test/zoo/chronos/model/test_prophet.py [hunk 4]
```diff
@@ -36,54 +36,61 @@ def setup_method(self, method):
...
 
         with pytest.raises(Exception,
                            match="Needs to call fit_eval or restore first before calling evaluate"):
-            self.model.evaluate(x=None, target=self.target)
+            self.model.evaluate(target=self.validation_data, data=None)
```
Comment (shane-huang) [inline pyzoo/test/zoo/chronos/model/test_prophet.py:82]: remove data=None

File: pyzoo/test/zoo/chronos/model/test_prophet.py [hunk 5]
```diff
@@ -36,54 +36,61 @@ def setup_method(self, method):
...
+            self.model.predict(data=1)
 
-        with pytest.raises(ValueError, match="We don't support input x currently"):
-            self.model.evaluate(x=1, target=self.target)
+        with pytest.raises(ValueError, match="We don't support input data currently"):
+            self.model.evaluate(target=self.validation_data, data=1)
 
         with pytest.raises(ValueError, match="Input invalid target of None"):
-            self.model.evaluate(x=None, target=None)
+            self.model.evaluate(target=None, data=None)
```
Comment (shane-huang) [inline pyzoo/test/zoo/chronos/model/test_prophet.py:74]: remove data=None

File: pyzoo/zoo/chronos/model/prophet.py [hunk 6]
```diff
@@ -24,100 +24,91 @@
...
             self._build(**config)
+            self.model_init = True
 
-        x = data['x']
-        target = data['val_y']
-        self.model.fit(x)
-        val_metric = self.evaluate(x=None, target=target, metrics=[self.metric])[0].item()
+        self.model.fit(data)
+        val_metric = self.evaluate(target=validation_data,
+                                   data=None,
```
Comment (shane-huang) [inline pyzoo/zoo/chronos/model/prophet.py:69]: remove data=None

**Comment by PeterQiu0516** [comment]
Jenkins passed: http://10.239.176.111:18888/view/ZOO-PR/job/ZOO-PR-Validation/6021/

**Pull request merged**